src/schema/quo.cljs

```diff
@@ -2,27 +2,75 @@
   (:require
     [schema.registry :as registry]))
 
-(def ^:private ?profile-picture-fn-params
+(def ^:private ?profile-picture-options
   [:map
    [:length :int]
    [:full-name :string]
    [:font-size :int]
    [:indicator-size {:optional true} [:maybe :int]]
    [:indicator-color {:optional true} [:maybe :string]]
    [:indicator-center-to-edge {:optional true} [:maybe :int]]
-   [:override-theme :schema.common/theme]
+   [:theme {:optional true} [:maybe :schema.common/theme]]
    [:background-color :string]
    [:color :string]
    [:size :int]
    [:ring? :boolean]
    [:ring-width :int]])
 
+(def ^:private ?account-image-uri-options
+  [:map
+   [:port :int]
+   [:ratio :double]
+   [:key-uid :string]
+   [:image-name :string]
+   [:theme :schema.common/theme]
+   [:override-ring? [:maybe :boolean]]])
+
+(def ^:private ?initials-image-uri-options
+  [:map
+   [:port :int]
+   [:ratio :double]
+   [:uppercase-ratio :double]
+   [:font-file :string]
+   [:theme :schema.common/theme]
+   [:key-uid {:optional true} [:maybe :string]]
+   [:public-key {:optional true} [:maybe :string]]
+   [:override-ring? {:optional true} [:maybe :boolean]]])
+
+(def ^:private ?contact-image-uri-options
+  [:map
+   [:port :int]
+   [:clock :int]
+   [:ratio :double]
+   [:image-name :string]
+   [:public-key :string]
+   [:theme :schema.common/theme]
+   [:override-ring? [:maybe :boolean]]])
+
+(def ^:private ?image-uri-config
+  [:or
+   [:map
+    [:kind [:enum :contact]]
```

Comment:
The word `:kind` feels a little off. I'd suggest `:type`. We use the `:type` keyword in other components too.

Reply:
+1 to renaming to :type. 

Also you could use the `:multi` schema here, which is useful when you have different combinations of schemas e.g.

```clj
[:merge [:map [:type [:enum :contact :account ...]]]
 [:multi {:dispatch :type}
  [:contact [:map 
    [:options ?contact-image-uri-options]
  [:account [:map ...]]]]]
```